src/wx/widgets/sdljoy.cpp

```diff
@@ -224,13 +222,11 @@ void wxSDLJoy::Poll()
             {
                 auto joy = e.cdevice.which;
```

Comment:
This should probably be e.jdevice.which

Reply:
Same comment to all other JOYDEVICE enums

Reply:
Valid point.

Reply:
Fixed, thank you.